e2etest/test-filters.yaml

```diff
@@ -0,0 +1,40 @@
+# This file defines which test patterns to skip based on environment configuration
+test_filters:
```

Comment:
nit: I think we can drop this (`filters = config["test_filters"]` seems odd when the file is called test-filters)